configs/openmrs_config/concepts/management.csv

```diff
@@ -3,4 +3,7 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:k
 a085b404-2596-48d6-a454-cd164d4a83fc,,,Monitor every 6 months (20% to <30%),ធ្វើការតាមដានរៀងរាល់៦ខែម្តង (ហានិភ័យ 20% ទៅ <30%),Monitor every 6 months (20% to <30%),ធ្វើការតាមដានរៀងរាល់៦ខែម្តង (ហានិភ័យ 20% ទៅ <30%),,,Misc,N/A,,,,
 9b305de5-9df6-4db9-820d-9a24541418ab,,,Monitor every 3 months (>=30%),ធ្វើការតាមដានរៀងរាល់៣ខែម្តង (ហានិភ័យ >=30%),Monitor every 3 months (>=30%),ធ្វើការតាមដានរៀងរាល់៣ខែម្តង (ហានិភ័យ >=30%),,,Misc,N/A,,,,
 7a67e555-1320-4197-a5df-54be3e68a1fe,,,Follow-up screening CVD risk factors,តាមដានពិនិត្យរុករកកត្តាហានិភ័យជំងឺសរសៃឈាម-បេះដូង,Follow-up screening CVD risk factors,តាមដានពិនិត្យរុករកកត្តាហានិភ័យជំងឺសរសៃឈាម-បេះដូង,,,Question,Coded,Monitor every year (<20%);Monitor every 6 months (20% to <30%);Monitor every 3 months (>=30%),,,
+4d32413a-95a4-45bf-95a0-ff81e26cfd35,,,CVD Risk Factors,,CVD Risk Factors,,,,Misc,N/A,,,,
+33f058fd-8341-4ca1-b843-be71e5fe59ac,,,Diabetes mellitus/Hypertension artérielle,,DM / HTA,,,,Misc,N/A,,,,
```

Comment:
I'm surprised of the "Hypertension artérielle" written in French like that. Where does that come from?

Reply:
`DM / HTA` -> `DM/HTA`

Reply:
@rbuisson, could you let me know the right spelling for the full name of `HTA` ?

Reply:
@kdaud if it is Hypertension the translation is in this Google sheet [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=68:68)